package.json

```diff
@@ -14,9 +14,9 @@
     "asar": "^0.13.0",
     "bower": "^1.3.12",
     "electron": "^1.6.1",
-    "electron-builder": "^17.0.1",
+    "electron-builder": "^18.8.1",
     "electron-icon-maker": "^0.0.3",
-    "electron-publisher-s3": "^17.0.2",
+    "electron-publisher-s3": "^19.0.1",
```

Comment:
I spent a little bit of time trying to find a changelog for this package and `electron-updater` but I could only find the overall `electron-builder` changlog. Did you find a different changelog? What are the changes we're picking up with these upgrades?